### DISCUSSION TITLE
Plans for asyncio-compliant interface?

### DISCUSSION THREAD

**Discussion by abaisero** [aatikturk/obsws-python#64] [Q&A]
Since websockets are pure I/O, there would be significant advantages in providing an async-compliant interface for asynchronous scripts and programs.  Has this been considered?

**Comment by onyx-and-iris** [comment]
Hi, thanks for raising this issue. Yes we have discussed this topic but neither of us have much time spent with the asyncio api, so while it would be a valuable addition, it wouldn't make sense for either of us two to implement this.

If you (or anyone else) wants to discuss this further, feel free to email me your discord, my email is visible on my profile. I would only want to implement this if it's fully functioning, tested and doesn't interfere with the current codebase.

There is currently an async-compliant interface [simpleobsws ](https://github.com/IRLToolkit/simpleobsws/tree/master) which I would encourage you to check out.
